source/includes/data-formats/bson.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <chrono>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main() {
+    mongocxx::instance instance;
+    mongocxx::uri uri("<connection string>");
+    mongocxx::client client(uri);
+
+    {
+        // Creates a BSON document using the list builder
+        // start-bson-list
+        bsoncxx::builder::list course_doc = { "title", "Poetry",
+                                              "department", "English" };
+
+        bsoncxx::builder::list courses_array = { "Poetry", "Literature",
+                                                 "Creative Writing" };
+
+        bsoncxx::document::value course{course_doc.extract()};
```

Comment:
```suggestion
        bsoncxx::document::value course{course_doc.view().get_document()};
```

To fix compile error:
```
/Users/kevin.albertson/code/cxx-bootstrap/investigations/ping/main.cpp:27:48: error: no member named 'extract' in 'bsoncxx::v_noabi::builder::list'
    bsoncxx::document::value course{course_doc.extract()};
```